modules/system/classes/UpdateManager.php

```diff
@@ -5,27 +5,19 @@
 use Carbon\Carbon;
 use Cms\Classes\ThemeManager;
 use Exception;
-use Illuminate\Console\OutputStyle;
-use Illuminate\Console\View\Components\Error;
-use Illuminate\Console\View\Components\Info;
 use Illuminate\Database\Migrations\DatabaseMigrationRepository;
 use Illuminate\Database\Migrations\Migrator;
 use Illuminate\Support\Facades\App;
-use Illuminate\Support\Facades\Cache;
-use Illuminate\Support\Facades\Lang;
-use RecursiveDirectoryIterator;
-use RecursiveIteratorIterator;
+use Mix\TestA\Plugin;
```

Comment:
Was the wrong class imported here?

Reply:
Yes, see: https://github.com/wintercms/winter/pull/1259/commits/16a4841c4c9bccd216c1da9770a34b6f4e9d6fc0